packages/build/src/telemetry/main.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+const { platform, version: nodeVersion } = require('process')
+
+const execa = require('execa')
+const osName = require('os-name')
+const { v4: uuidv4 } = require('uuid')
+
+const { version } = require('../../package.json')
+const { roundTimerToMillisecs } = require('../time/measure')
+
+// Script file that will be responsible for making the actual telemetry request
+const REQUEST_FILE = `${__dirname}/request.js`
+
+// Send telemetry request when build completes
+const trackBuildComplete = async function ({
+  status,
+  commandsCount,
+  netlifyConfig,
+  durationNs,
+  siteInfo,
+  telemetry,
+  testOpts,
+}) {
+  if (!telemetry) {
+    return
+  }
+
+  const payload = getPayload({ status, commandsCount, netlifyConfig, durationNs, siteInfo })
```

Comment:
Should this statement be covered by the `try` too, just in case?

Reply:
Yep, addressed in https://github.com/netlify/build/pull/2349/commits/2bca6a408b5f9a39ba5810407b643ba66f1d19ec